test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1293,14 +1293,20 @@ private static synchronized void showUI() {
 
 
     /**
-     * Forcibly pass the test.
+     * Forcibly pass the test. This should be used in semi-automatic tests when
+     * the test determines that all the conditions for passing the test are met.
+     * This should not be used in cases where a resource is unavailable or a
```

Comment:
```suggestion
     * <p>
     * This method should not be used in cases where a resource is unavailable or a
```
Again imperative could better.